07.ls_object/display.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+class Display
+  def initialize(entries)
+    @entries = entries
+  end
+
+  def display_entries(long_option)
```

Comment:
```suggestion
  def display_entries(long_format = false)
```

lsコマンドのデフォルトがlongじゃない方なので、メソッドも同じくデフォルトはfalseにしておくとか。
あと引数の名前も、表示形式の制御に使うんですよ、ということがわかるように format を含めた方がいいかなと思いました。

Reply:
`def display_entries(long_format: false)`に変更しました。